stephan-livera-podcast/2023-01-14-james-a-new-way-to-hodl.md

```diff
@@ -0,0 +1,291 @@
+---
```

Comment:
file name should be stephan-livera-podcast/2023-01-14-james-obeirne-a-new-way-to-hodl.md

Reply:
the filename is renamed.